test/Integration/IntegrationTestBase.cs

```diff
@@ -76,6 +76,7 @@ public IntegrationTestBase(ITestOutputHelper output = null)
             this.TestOutput = output;
             this.SetupDatabase();
             this.StartAzurite();
+            this.BuildJavaFunctionApps();
```

Comment:
We shouldn't do this for every test class - you can add it to the test fixture I'm adding in https://github.com/Azure/azure-functions-sql-extension/pull/527 once that's in